Bottom: a03b02da89a7ade273aa11fb0b1b6e9fba5f0670
Top:    2f1d8ebd4f00d13499cabb38ab2cea506bca544f
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 14:37:02 +0800

autofs-5.0.7 - fix incorrect value reference in parse_line()

This chack should clearly be on the contents of key not the pointer value.


---

diff --git a/lib/defaults.c b/lib/defaults.c
index 1e89509..7c65387 100644
--- a/lib/defaults.c
+++ b/lib/defaults.c
@@ -167,7 +167,7 @@ static int parse_line(char *line, char **res, char **value)
 	while (*key && *key == ' ')
 		key++;
 
-	if (!key)
+	if (!*key)
 		return 0;
 
 	if (!(val = strchr(key, '=')))
